alpha/lib/model/LiveEntry.php

```diff
@@ -14,6 +14,11 @@ abstract class LiveEntry extends entry
 	const SIMULIVE_CAPABILITY = 'simulive_capability';
 	const LOW_LATENCY_TAG = 'lowlatency';
 
+	const LIVE_BACKEND_PARAM = 'live_backend';
+	const BROADCAST_MAP = 'broadcast';
+	const LIVE_BACKEND_WOWZA = 0;
```

Comment:
Why not use string constants like "wowze" & "liveNG" this way you won't have to crosscheck config value 0 to the class to see what it is.